image/CMakeLists.txt

```diff
@@ -19,10 +20,10 @@ multicomp_install(
     COMPONENTS Standalone Update
 )
 
-multicomp_install(
-    DIRECTORY
-        ${ASSETS_DEST_DIR}/current/
-        ${ASSETS_DEST_DIR}/user
-    DESTINATION ./
-    COMPONENTS Standalone Update
-)
+# multicomp_install(
+#     DIRECTORY
+#         ${ASSETS_DEST_DIR}/current/
+#         ${ASSETS_DEST_DIR}/user
+#     DESTINATION ./
+#     COMPONENTS Standalone Update
+# )
```

Comment:
Why was it commented out? Remove it completely or at least leave a comment explaining why.

Reply:
ti was temporarily commented out, it's removed :) similarly in the comment below ;) tx